packages/sanity/src/router/RouterProvider.tsx

```diff
@@ -52,16 +52,8 @@ export interface RouterProviderProps {
  * @public
  */
 export function RouterProvider(props: RouterProviderProps): React.ReactElement {
-  // TODO: can we do nested routes?
```

Comment:
Not yet, it requires a pretty significant refactor that require a lot of parallel work in the Structure Builder internals 😭